crates/swc/tests/tsc-references/deleteChain.1.normal.js

```diff
@@ -1,37 +1,15 @@
 //// [deleteChain.ts]
```

Comment:
https://www.typescriptlang.org/play?ts=4.8.0-beta#code/PTAEAEGcBcCcEsDG0Bco4FcCmAoHATLRAGwENYtREB7AOxlGoEY0NbCAzeWrfUAH1ABvUACM0MBLQDmoAL4BuAlmJZolZgH4AdKKWFV60AAotugJRLlJcpRr1ojAEyt2WLjz6CR44VQlw3LJy8voqahpOOqLaiGGGlKZRurGWeIQ2FFR0DNQAzGg+rpzcvAJ+iAFSwaHKCYx5ujpxdREm+U2pVhlkWfa5ACyFYprDlX74o35YVUHy8yGKrUbUA52I2pPaWPFtpmuizZvmOjvLiavrm6dp1r12OY7UAKzDh8bmY8OTwzOgknNFgtditnp0PrFrtsQRcwYdtBCNlssLcerZsg5GAA2N5TETjEQ-aazGQLYHnbGdADaAHJEDSALpQ2lYRkw9pY6l0xnMmmshmWIA
Typescript still emits some errors, but not `The operand of a delete operator must be a property reference.`
It is related to 
```ts
 function checkDeleteExpressionMustBeOptional(expr: AccessExpression, symbol: Symbol) {
    const type = getTypeOfSymbol(symbol);
    if (strictNullChecks &&
        !(type.flags & (TypeFlags.AnyOrUnknown | TypeFlags.Never)) &&
        !(exactOptionalPropertyTypes ? symbol.flags & SymbolFlags.Optional : getTypeFacts(type) & TypeFacts.IsUndefined)) {
        error(expr, Diagnostics.The_operand_of_a_delete_operator_must_be_optional);
    }
}
```

Reply:
It is related to type information, so I think we could ignore them. As Babel did the same thing. https://babeljs.io/repl/#?browsers=defaults%2C%20not%20ie%2011%2C%20not%20ie_mob%2011&build=&builtIns=false&corejs=3.21&spec=false&loose=false&code_lz=PTAEAEGcBcCcEsDG0Bco4FcCmAoHATLRAGwENYtREB7AOxlGoEY0NbCAzeWrfUAH1ABvUACM0MBLQDmoAL4BuAlmJZolZgH4AdKKWFV60AAotugJRLlJcpRr1ojAEyt2WLjz6CR44VQlw3LJy8voqahpOOqLaiGGGlKZRurGWeIQ2FFR0DNQAzGg-rpzcvAJ-iAFSwaHKCYx5ujpxdREm-U2pVhlkWfa5ACyFYprDlX74o35YVUHy8yGKrUbUA52I2pPaWPFtpmuizZvmOjvLiavrm6dp1r12OY7UAKzDh8bmY8OTwzOgknNFgtditnp0PrFrtsQRcwYdtBCNlssLcerZsg5GAA2N5TETjEQ_aazGQLYHnbGdADaAHJEDSALpQ2lYRkw9pY6l0xnMmmshmWIA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env%2Ctypescript&prettier=false&targets=&version=7.18.13&externalPlugins=&assumptions=%7B%7D